node/pkg/governor/mainnet_chains.go

```diff
@@ -10,21 +10,21 @@ import (
 
 func chainList() []chainConfigEntry {
 	return []chainConfigEntry{
-		chainConfigEntry{emitterChainID: vaa.ChainIDSolana, dailyLimit: 50000000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDEthereum, dailyLimit: 50000000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDTerra, dailyLimit: 500000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDBSC, dailyLimit: 5000000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDPolygon, dailyLimit: 5000000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDAvalanche, dailyLimit: 5000000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDOasis, dailyLimit: 500000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDAlgorand, dailyLimit: 200000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDAurora, dailyLimit: 500000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDFantom, dailyLimit: 500000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDKarura, dailyLimit: 500000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDAcala, dailyLimit: 500000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDKlaytn, dailyLimit: 500000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDCelo, dailyLimit: 500000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDNear, dailyLimit: 200000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDTerra2, dailyLimit: 500000},
+		chainConfigEntry{emitterChainID: vaa.ChainIDSolana, dailyLimit: 50_000_000, bigTransactionSize: 5_000_000},
+		chainConfigEntry{emitterChainID: vaa.ChainIDEthereum, dailyLimit: 50_000_000, bigTransactionSize: 5_000_000},
+		chainConfigEntry{emitterChainID: vaa.ChainIDTerra, dailyLimit: 500_000, bigTransactionSize: 50_000},
+		chainConfigEntry{emitterChainID: vaa.ChainIDBSC, dailyLimit: 5_000_000, bigTransactionSize: 500_000},
+		chainConfigEntry{emitterChainID: vaa.ChainIDPolygon, dailyLimit: 5_000_000, bigTransactionSize: 500_000},
+		chainConfigEntry{emitterChainID: vaa.ChainIDAvalanche, dailyLimit: 5_000_000, bigTransactionSize: 500_000},
+		chainConfigEntry{emitterChainID: vaa.ChainIDOasis, dailyLimit: 500_000, bigTransactionSize: 50_000},
+		chainConfigEntry{emitterChainID: vaa.ChainIDAlgorand, dailyLimit: 200_000, bigTransactionSize: 20_000},
+		chainConfigEntry{emitterChainID: vaa.ChainIDAurora, dailyLimit: 500_000, bigTransactionSize: 50_000},
+		chainConfigEntry{emitterChainID: vaa.ChainIDFantom, dailyLimit: 500_000, bigTransactionSize: 50_000},
+		chainConfigEntry{emitterChainID: vaa.ChainIDKarura, dailyLimit: 500_000, bigTransactionSize: 50_000},
+		chainConfigEntry{emitterChainID: vaa.ChainIDAcala, dailyLimit: 500_000, bigTransactionSize: 50_000},
+		chainConfigEntry{emitterChainID: vaa.ChainIDKlaytn, dailyLimit: 500_000, bigTransactionSize: 50_000},
+		chainConfigEntry{emitterChainID: vaa.ChainIDCelo, dailyLimit: 500_000, bigTransactionSize: 50_000},
+		chainConfigEntry{emitterChainID: vaa.ChainIDNear, dailyLimit: 200_000, bigTransactionSize: 20_000},
+		chainConfigEntry{emitterChainID: vaa.ChainIDTerra2, dailyLimit: 500_000, bigTransactionSize: 50_000},
```

Comment:
Don't we need to add `checkForBigTransactions: true`? for big transactions to be recognized by `isBigTransfer`?

Reply:
Nevermind, this is being set properly.